content/ch-applications/vqe-molecules.ipynb

```diff
@@ -361,7 +361,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Assume the depth setting is set to $d$. Then, RyRz has $n\\times (d+1)\\times 2$ parameters, Ry with linear entanglement has $2n\\times(d + \\frac{1}{2})$ parameters, and Ry with full entanglement has $d\\times n\\times \\frac{(n + 1)}{2} + n$ parameters."
+    "Assume the depth setting is set to $d$. Then, RyRz has $2n\\times (d+1)$ parameters and Ry has $n\\times (d+1)$ parameters."
```

Comment:
```suggestion
    "Assume the depth setting is set to $d$. Then, RyRz has $2n\\times (d+1)$ parameters and Ry (not shown) has $n\\times (d+1)$ parameters."
```
Sorry- another thing: I think we should mention Ry is not displayed above to avoid the same confusion

Reply:
Ok done